modules/basics/src/test/java/com/opengamma/basics/CurrencyAmountAssert.java

```diff
@@ -0,0 +1,119 @@
+/**
+ * Copyright (C) 2014 - present by OpenGamma Inc. and the OpenGamma group of companies
+ *
+ * Please see distribution for license.
+ */
+package com.opengamma.basics;
+
+import org.assertj.core.api.AbstractAssert;
+import org.assertj.core.api.Assertions;
+import org.assertj.core.data.Offset;
+
+import com.opengamma.basics.currency.Currency;
+import com.opengamma.basics.currency.CurrencyAmount;
+import com.opengamma.collect.result.Result;
+
+/**
+ * An assert helper that provides useful AssertJ assertion
+ * methods for {@link Result} instances.
```

Comment:
This looks like it was copy-pasted from ``Result`` and some of the references to ``Result`` haven't been updated to ``CurrencyAmount``.